meinberlin/apps/moderatorremark/templates/meinberlin_moderatorremark/includes/popover_remark.html

```diff
@@ -12,19 +12,19 @@
         id="idea-remark__dropdown"
     >
         <i class="fa fa-comment" aria-label="{% trans 'Moderator Remark' %}"></i>
-        <i class="fa fa-check-circle idea-remark__btn__notify {{ remark|yesno:",d-none" }}" aria-label="Remark-Notification"></i>
+        <i class="fa fa-check-circle idea-remark__btn__notify {{ remark|yesno:",d-none" }}" aria-hidden="true"></i>
     </button>
-    <ul class="dropdown-menu idea-remark" id="remark__dropdown-menu" aria-labelledby="idea-remark">
+    <ul class="dropdown-menu idea-remark" id="remark__dropdown-menu">
         <li>
             <h3>{% trans 'Remark' %}</h3>
             <form class="general-form" id="idea-remark__form">
                 <div class="form-group idea-remark__text">
-                    <textarea class="form-control" id="idea-remark__input" placeholder="{% trans 'Write down your remark!' %}"></textarea>
+                    <textarea class="form-control" id="idea-remark__input" aria-label="moderator-remark-input" placeholder="{% trans 'Write down your remark!' %}"></textarea>
```

Comment:
Did you find anything about labeling a text-area? I only found this https://www.w3.org/TR/aria-in-html/#fifth, but we don't have a label, but a placeholder and a form-hint. Let's find an example in the forms in mB!